pkg/controller/openpitrix/helmrelease/helm_release_controller.go

```diff
@@ -110,6 +113,24 @@ func (r *ReconcileHelmRelease) Reconcile(request reconcile.Request) (reconcile.R
 		// The object is not being deleted, so if it does not have our finalizer,
 		// then lets add the finalizer and update the object.
 		if !sliceutil.HasString(instance.ObjectMeta.Finalizers, HelmReleaseFinalizer) {
+			clusterName := instance.GetRlsCluster()
+			if clusterName != "" {
+				clusterInfo, err := r.clusterLister.Get(clusterName)
+				if err != nil {
+					if apierrors.IsNotFound(err) {
+						//cluster not exists, delete this helmRelease
```

Comment:
Do nothing?

Reply:
Yes.  If cluster was deleted, we just delete the crd and  left helm deployments on the member cluster